stream_alert_cli/outputs/helpers.py

```diff
@@ -58,28 +62,25 @@ def kms_encrypt(region, data, kms_key_alias):
         str: Encrypted ciphertext data blob
     """
     try:
-        client = boto3.client('kms', region_name=region)
-        response = client.encrypt(KeyId='alias/{}'.format(kms_key_alias),
-                                  Plaintext=data)
-        return response['CiphertextBlob']
+        return AwsKms.encrypt(data, region=region, key_alias=kms_key_alias)
     except ClientError:
         LOGGER.error('An error occurred during credential encryption')
         raise
 
+
 def send_creds_to_s3(region, bucket, key, blob_data):
     """Put the encrypted credential blob for this service and destination in s3
 
+    FIXME (derek.wang) This method is deprecated and PROBABLY not being used anywhere anymore.
+
     Args:
         region (str): AWS region to use for boto3 client
         bucket (str): The name of the s3 bucket to write the encrypted credentials to
         key (str): ID for the s3 object to write the encrypted credentials to
         blob_data (bytes): Cipher text blob from the kms encryption
     """
     try:
-        client = boto3.client('s3', region_name=region)
-        client.put_object(Body=blob_data, Bucket=bucket, Key=key)
-
-        return True
+        return AwsS3.put_object(blob_data, bucket=bucket, key=key, region=region)
```

Comment:
nice changes - if this function (`send_creds_to_s3`) is in fact not being anywhere else, you can freely remove it